src/temporal/t.rast.aggregate.seasons/t.rast.aggregate.seasons.html

```diff
@@ -12,14 +13,14 @@ <h2>DESCRIPTION</h2>
 <p>
 
 <h2>EXAMPLES</h2>
-Calculate seasonal data from
+Calculate seasonal data from an input space time raster datasets with an output space time raster datasets
```

Comment:
```suggestion
Calculate seasonal data from an input space time raster dataset
```

Reply:
Probably the better change is "datasets with an output space time raster datasets" -> "dataset with a unified output space time raster dataset"

Reply:
yes, I like that. But then the manual should explain that if the output option is not used, the result is not a strds. Is that correct?

Reply:
No, right now without `output` option it create a singular space time raster dataset for each year of the input space time raster dataset. I'm going to improve documentation.

Reply:
Fixed in the new pull request #1010